common/app/routes/Settings/settings.less

```diff
@@ -88,7 +101,7 @@
 
 .danger-zone-panel {
   background-color: #fff;
-
+  
```

Comment:
Please remove the extra white space

Reply:
@Bouncey Done.
 
Btw I added it because in `settings.less` before every child-style-declaration, there was a carriage return.